### ISSUE TITLE
Poor visibility on landscape video view.

### ISSUE THREAD

**Issue by BeyondSpace609520** [SimpleMobileTools/Simple-Gallery#1868]
![Screenshot_1592386848](https://user-images.githubusercontent.com/56730274/84882380-7819f380-b08f-11ea-8064-2faf69643089.png)

I can't see elapsed time textview on landscape.

**Comment by tibbi** [comment]
is that the latest app version too? Those top and bottom bars should be gradient, not pure black

**Comment by BeyondSpace609520** [comment]
Sorry, I have customized some code. But it is no matter for this problem.
Navigation Bar is over on elapsed time textView.

**Comment by tibbi** [comment]
right, I see. It happens only if the navigation bar is on the left side, not if on the right.
